plugins/react-native.js

```diff
@@ -69,7 +69,12 @@ function reactNativePlugin(Raven) {
         }
     });
 
-    ErrorUtils.setGlobalHandler(Raven.captureException.bind(Raven));
+    var defaultHandler = (ErrorUtils.getGlobalHandler && ErrorUtils.getGlobalHandler())
+     || ErrorUtils._globalHandler;
+    ErrorUtils.setGlobalHandler(function(...args){
+      defaultHandler(...args);
+      Raven.captureException(...args);
```

Comment:
We should not pass arbitrary / unknown arguments to `captureException` ... I think declaring the arguments in `setGlobalHandler`, and just passing the first argument (the error) is fine:

```javascript
Raven.captureException(error);
```